config/config.go

```diff
@@ -49,11 +49,13 @@ func newDefaultCluster() *Cluster {
 		TLSCertDurationDays:      365,
 		ContainerRuntime:         "docker",
 		ControllerCount:          1,
+		ControllerCreateTimeout:  "PT15M",
 		ControllerInstanceType:   "m3.medium",
 		ControllerRootVolumeType: "gp2",
 		ControllerRootVolumeIOPS: 0,
 		ControllerRootVolumeSize: 30,
 		WorkerCount:              1,
+		WorkerCreateTimeout:      "PT15M",
```

Comment:
15 minutes is just a really lenient timeout and should be sufficient of nodes of any type (even t2's). If people want to specify their own timeouts (and fail fast! :+1:) they can.